### PR TITLE
Brush up release flow and write current version explicitly.

### PR DESCRIPTION
**Pull request by kazuki-ma**
Current version = `3.2.0-SNAPSHOT`.

Formatted version for review: https://github.com/line/line-bot-sdk-java/blob/kazuki-ma/release_flow/README.md

### CODE REVIEW

**Comment by kazuki-ma** [comment]
I'll define task for releases in another PR.
 https://github.com/line/line-bot-sdk-java/issues/424 filed.

File: README.md [hunk 1]
```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
 
 Please check [CONTRIBUTING](CONTRIBUTING.md) before making a contribution.
 
+## Release flow
+
+As of 2019-12, releaser is LINE's employee.
```
Comment (imasahiro) [inline README.md]: I'm pretty much not sure this line is needed or not.

File: README.md [hunk 2]
```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
...
+
+Basically, versions defined in `gradle.properties`.
```
Comment (imasahiro) [inline README.md]: 0. Confirm line-bot-sdk-java version is defined at `/gradle.properties`.

File: README.md [hunk 3]
```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
...
+
+1. Before update, update `gradle.properties` to remove `-SNAPSHOT`. e.g (`version=3.2.0`)
```
Comment (imasahiro) [inline README.md]: 1. Before release, updates a `version` property at `gradle.properties` with release version (without SNAPSHOT), for example `version=3.2.0`.

File: README.md [hunk 4]
```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
...
+2. Commit `gradle.properties` and tag version name. example: `git add -u && git commit -m 'Release 3.2.0' && git tag '3.2.0' && git push`
```
Comment (imasahiro) [inline README.md]: Commits `gradle.properties` and make a tag with version that specified at step1.
For example, 
```sh
$ git add gradle.properties
$ git commit -m "Release 3.2.0"
$ git tag '3.2.0'
$ git push upstream
```

File: README.md [hunk 5]
```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
...
+3. Proceeding release action (upload central etc.)
```
Comment (imasahiro) [inline README.md]: 3. Upload the artifacts to the staging repository
4. Close and release the staging repository at https://oss.sonatype.org/

File: README.md [hunk 6]
```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
...
+4. Update `gradle.properties` to `NEXT.GREAT.VERSION-SNAPSHOT`
+5. `git commit -m 'Start NEXT.GREAT.VERSION(-SNAPSHOT)'` and `git push`.
```
Comment (imasahiro) [inline README.md]: 6. Announce the release via Twitter.

File: README.md (hunk 1 above)
Comment (kazuki-ma) [reply README.md]: Exactly. Let me delete it.

File: README.md [hunk 7]
```diff
@@ -127,6 +127,23 @@ See http://semver.org/.
...
+    For example,
+    ```sh
+    $ git add gradle.properties
+    $ git commit -m "Release 3.2.0"
+    $ git tag '3.2.0'
+    $ git push upstream
+    ```
+3. Upload the artifacts to the staging repository
+4. Close and release the staging repository at https://oss.sonatype.org/
+5. Update `gradle.properties` to `NEXT.GREAT.VERSION-SNAPSHOT`
```
Comment (imasahiro) [inline README.md]: Updates version property at `gradle.properties` to next version with `SNAPSHOT`, for example `version=3.2.1-SNAPSHOT`

File: README.md [hunk 8]
```diff
@@ -127,6 +127,23 @@ See http://semver.org/.
...
+6. `git commit -m 'Start NEXT.GREAT.VERSION(-SNAPSHOT)'` and `git push`.
```
Comment (imasahiro) [inline README.md]: Commits `gradle.properties` and push to upstream repository
For example,
```sh
$ git commit -m "Start NEXT.GREAT.VERSION-SNAPSHOT"
$ git push upstream
```

**Comment by imasahiro** [comment]
ping gently

File: README.md [hunk 9]
```diff
@@ -127,6 +127,23 @@ See http://semver.org/.
...
+## Release flow
+
+0. Confirm line-bot-sdk-java version is defined at `./gradle.properties`.
+1. Before release, updates a version property at `gradle.properties` with release version (without `SNAPSHOT`), for example `version=3.2.0`.
+2. Commits `gradle.properties` and make a tag with version that specified at step1.
+    For example,
+    ```sh
+    $ git add gradle.properties
+    $ git commit -m "Release 3.2.0"
+    $ git tag '3.2.0'
```
Comment (imasahiro) [inline README.md:139]: Please make a pull request than creating a tag.

**Pull request closed without merging**